src/content/app/new-species-selector/components/species-search-field/AddFoundSpecies.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+
+import ShadedInput from 'src/shared/components/input/ShadedInput';
+import { PrimaryButton } from 'src/shared/components/button/Button';
+import TextButton from 'src/shared/components/text-button/TextButton';
+
+import styles from './SpeciesSearchField.scss';
+
+export type Props = {
+  query: string;
+  canAdd: boolean;
+  onAdd: () => void;
+  onCancel: () => void;
+};
+
+const AddFoundSpecies = (props: Props) => {
```

Comment:
Won't `AddSelectedSpecies` be more self descriptive?

Reply:
It's that word again — selected — I am never quite sure what it means in the context of our site.

Reply:
So how about `AddStagedSpecies` instead?

Reply:
I suppose I can change it to `AddSpecies` (or `AddGenomes`).

Reply:
Renamed `AddFoundSpecies`  to `AddSpecies`